### ISSUE TITLE
Use snake case in configuration

### ISSUE THREAD

**Issue by Fabrn** [endroid/qr-code-bundle#81]
Hello there !

Using version 5.0.1 of the bundle, the compiler crashes because of the fact that previously, the configuration of the bundle was handling snake case. However, as we can see in the `EndroidQrCodeExtension::createBuilderDefinition` method, snake case is not handled anymore and the configuration is required to be camel case, which is weird considering the fact that the bundle name must be in snake case and snake case is also used as a norm for every bundle.

My configuration looks like this : 

```yaml
endroid_qr_code:
    writer: 'Endroid\QrCode\Writer\PngWriter'
    size: 300
    margin: 10
    errorCorrectionLevel: 'low'
    validateResult: false
```

I personally think that this is an issue and the configuration should be handled as snake case.

What do you guys think about that ?

Thank you guys and have a great day 😃

**Comment by endroid** [comment]
Hi @Fabrn I agree that snake case is more conventional for configuration. Will look at it later, thank you!

**Comment by Fabrn** [comment]
Hi ! Thanks for your quick review 🙂

I might help you on that if you wish !
Anyway thank you for considering a fix !

**Comment by endroid** [comment]
Fixed in 5.0.2
